src/Standards/Squiz/Sniffs/WhiteSpace/MemberVarSpacingSniff.php

```diff
@@ -86,22 +102,34 @@ protected function processMemberVar(File $phpcsFile, $stackPtr)
 
         $prev       = $phpcsFile->findPrevious(Tokens::$emptyTokens, ($first - 1), null, true);
         $foundLines = ($tokens[$first]['line'] - $tokens[$prev]['line'] - 1);
-        if ($foundLines === 1) {
+        $error      = 'Expected %s blank line before member var; %s found';
+        $expected   = $this->spacing;
+        $isFirst    = $tokens[$prev]['code'] === T_OPEN_CURLY_BRACKET;
+
+        // We allow specifying a different number of spaces to the first operator.
+        if ($isFirst === true) {
+            $expected = $this->firstMemberSpacing;
+            $error    = 'Expected %s blank lines before first member var; %s found';
+        }
+
+        if ($foundLines === $this->spacing || ($isFirst === true && $foundLines === $this->firstMemberSpacing)) {
```

Comment:
I got an error when I pass in zero as a parameter:

```xml
<property name="firstMemberSpacing" value="0" />
```
It seems to me a cast to int is missing:

```php
$foundLines === (int)$this->firstMemberSpacing
```